### ISSUE TITLE
Item count is not shown anywhere

### ISSUE THREAD

**Issue by DistrictZion** [TeamBroCode/ox_inventory-brocode#7]
As title says.

**Comment by AnishBplayz** [comment]
Was never in our design

**Comment by DistrictZion** [comment]
Is there a way to add it? or point me in the direction as to how.
Thanks in advance

**Comment by AnishBplayz** [comment]
Yes there is, you can self edit the design to add it for yourself, also item count is shown in the item tooltip so am not sure what you meant with "anywhere".

**Comment by DistrictZion** [comment]
"anywhere" as in, it was a typo, and so I edited. I meant., in the OG design. 
But as you said, it was not intended. 
Now, I do not know how to do it. So, could you tell me how. I'm not great at coding, so I needed help.
If you want to be counter argumentative, and not helpful, it's okay. 
I understand. I will figure it out myself. I was merely asking for an edit since it is very good, and I'm bad at coding. 
Thank you for the response. 

Great design though.
